profile/src/main/java/org/apache/karaf/profile/assembly/Builder.java

```diff
@@ -617,6 +608,15 @@ public String getValue(String key) {
         // Installed stage
         //
         installStage(installedProfile, allBootFeatures);
+
```

Comment:
For the record, this is already fixed on `master` and `karaf-4.1.x` branches.

Reply:
Ok, thats good. Still needed this part in to make my scenario work though.